tests/system/feature_store/test_feature_store.py

```diff
@@ -260,9 +264,7 @@ def _get_offline_vector(
         )
         resp = fstore.get_offline_features(
             vector,
-            entity_rows=trades.set_index(
-                "ticker"
-            ),  # test when the relation keys are indexes.
```

Comment:
Why is this removed? It doesn't seem related to this PR.

Reply:
right - will fix